colorcoder/Main.java

```diff
@@ -1,120 +1,13 @@
 package colorcoder;
 
```

Comment:
what is the value addition to the program when class name and method name is same....

Reply:
I used Main as the driver class thus adding no value the program itself. What could be the better approach in this case?